### DISCUSSION TITLE
有更详细的业界成功案例吗？

### DISCUSSION THREAD

**Discussion by xiawenqi** [antvis/X6#3283] [Q&A] [answered]
我们在做技术选型，目前首选X6， 想有一些更具体的使用X6的成功案例可以参考。

目前官网上有一些感谢信赖的ICON，但是没有使用详情。博客里找到了翼支付技术选型的过程，但是没有展示后续使用的情况。

**Comment by NewByVector** [comment, marked as answer]
我们内部有很多平台都是使用 X6，比如 [隐语开放平台](https://www.yuque.com/antv/x6/tr30g76v1x560cxr)，不过内部的一些平台暂时不好对外访问。

**Reply by xiawenqi** [reply]
官网的ICON里有个京东的案例，是什么，有详情或者链接吗

**Comment by Eve-Sama** [comment]
你想看什么内容呢 一般就是告诉你他们哪个模块用了x6, 详细实现是不会放出来的.

**Comment by xiawenqi** [comment]
主要是想看看有没有商用的成功案例，方便举例。隐语开放平台这个案例够用了， 感谢~
